awd/deposit.py

```diff
@@ -94,19 +101,30 @@ def deposit(
     date = datetime.today().strftime("%m-%d-%Y %H:%M:%S")
     s3_client = s3.S3()
     sqs_client = sqs.SQS()
+    dynamodb_client = dynamodb.DynamoDB()
     dois = crossref.get_dois_from_spreadsheet(doi_file_path)
+    doi_items = dynamodb_client.retrieve_doi_items_from_database(doi_table)
     for doi in dois:
+        if doi_to_be_added(doi, doi_items):
+            dynamodb_client.add_doi_item_to_database(doi_table, doi)
+        elif doi_to_be_retried(doi, doi_items) is False:
+            continue
+        dynamodb_client.update_doi_item_status_in_database(doi_table, doi, 1)
```

Comment:
Same as the other calls. Otherwise 1, 2, 3, 4 appear as special or magical numbers.